src/commonMain/kotlin/exchange.dydx.abacus/processor/wallet/account/AccountProcessor.kt

```diff
@@ -941,6 +951,14 @@ internal class V4AccountProcessor(parser: ParserProtocol) : BaseProcessor(parser
             payload?.get("totalTradingRewards")
         )
         modified.safeSet("tradingRewards", modifiedTradingRewards)
+
+
+        /* block trading rewards are only sent in subaccounts.0 channel */
```

Comment:
i think chris commented on the indexer spec that it'll be sent to the subaccount channel that received the fill